parquet-column/src/main/java/org/apache/parquet/column/values/dictionary/DictionaryValuesWriter.java

```diff
@@ -180,6 +191,19 @@ public Encoding getEncoding() {
   @Override
   public void reset() {
     encodedValues = new IntList();
+    for (RunLengthBitPackingHybridEncoder encoder : encoders) {
+      encoder.close();
+    }
+    encoders.clear();
+  }
```

Comment:
just confirming this was addressed, Github seems to think the comment is still active because the line numbers changed due to other edits in the file.